src/schema/v2/homeView/sections/ExploreByCategories.ts

```diff
@@ -0,0 +1,61 @@
+import { HomeViewSection } from "."
+import { HomeViewSectionTypeNames } from "../HomeViewSection"
+
+export const ExploreByCategories: HomeViewSection = {
+  id: "home-view-section-explore-by-categories",
+  type: HomeViewSectionTypeNames.HomeViewSectionMarketingCollectionCategories,
+  component: {
+    title: "Explore by categories",
+  },
+  requiresAuthentication: false,
+
+  resolver: () => categories,
+}
+
+// TODO: temporary images, we still need to wait for the assets and to incorporate the image versions too
+const categories = [
+  {
+    name: "Medium",
+    image: {
+      image_url: "https://files.artsy.net/images/capivara_chimarrao.jpg",
+    },
+    href: "/marketing-collections/Medium",
+  },
+  {
+    name: "Movement",
+    image: {
+      image_url: "https://files.artsy.net/images/capivara_chimarrao.jpg",
+    },
+    // TODO: once the Movement MarketingCollectionCategory is available, we can use it here
+    href: "/marketing-collections/Medium",
+  },
+  {
+    name: "Color",
+    image: {
+      image_url: "https://files.artsy.net/images/capivara_boia.jpg",
+    },
+    href: "/marketing-collections/Collect by Color",
+  },
+  {
+    name: "Size",
+    image: {
+      image_url: "https://files.artsy.net/images/capivara_nadando.jpg",
+    },
+    href: "/marketing-collections/Collect by Size",
+  },
+  {
+    name: "Price",
+    image: {
+      image_url: "https://files.artsy.net/images/capivara_filhotes.jpg",
+    },
+    href: "/marketing-collections/Collect by Price",
```

Comment:
**question**: Is this going to be a "page" in Eigen with the slug "Collect by Price"? If so, does that slug sounds correct?
An alternative would be to add another field for the `category: "Collect by Price"` and use an href like `/marketing-collections/collect-by-price`

Reply:
It will be a screen on Eigen with this exact slug, we do not use slugs to filter with `marketingCollections(category: "Collect by Size")`, so we need to use the name.

I am wiring Eigen locally to see if the router accepts this kind params with spaces